.circleci/config.yml

```diff
@@ -86,6 +86,13 @@ workflows:
           requires:
               - verify-code
           validation: true
+      - notify-about-main-update:
```

Comment:
should it be triggered only when we merge into main branch?

Reply:
Well we may also think about release branch updates, I suppose. Let me think.
cc @pengdev wdyt?

Reply:
yes, I think we need to automate the release branch cherry-picks as well. We can do it in as a follow-up.

Reply:
Yes, will do separately